lib/CheckerGranule.py

```diff
@@ -3,6 +3,7 @@
 import socket
 from datetime import *
 
+LINK_CHECK_TIMEOUT = 5
```

Comment:
this one has spaces around the equals, the one in the other file does not. I like the spaces better, but I don't know what's better for python and I think your preference should be favored here.